### PR TITLE
ARM endpoint and JSON Transformer Documentation

### PR DESCRIPTION
**Pull request by ericajlin**
Documentation for Azure Resource Manager (ARM) query provider and JSON Transformer

### CODE REVIEW

File: Documentation/Samples/AlertDataARM.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    3. Required: `Checked`
+    4. Get data from: `Default Subscriptions`
+    5. Use the `Save` button in the toolbar to save this parameter. 
+
+To get a list of existing alerts for a subscription, we will utilize the [Alerts - Get All REST call](https://docs.microsoft.com/en-us/rest/api/monitor/alertsmanagement/alerts/getall)
+
+3. Click `Add query` to create a query control.
+    1. Data source: `Azure Resource Manager (Preview)`
+    2. Http Method: `GET`
+    3. Path: `/subscriptions/{subscription:id}/providers/Microsoft.AlertsManagement/alerts`
```
Comment (yves-chan) [inline Documentation/Samples/AlertDataARM.md]: `subscription:id` should be `Subscription:id`

File: Documentation/Samples/AlertDataARM.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    4. Add the api-version parameter in the `Parameters` tab
+        1. Parameter: `api-version`
```
Comment (yves-chan) [inline Documentation/Samples/AlertDataARM.md]: might be good to add a note about where to find the latest api version

File: Documentation/Samples/Samples.md [hunk 3]
```diff
@@ -21,3 +21,9 @@
 ### Combining data from different sources using the merge control
 
 [![Combining data from different sources in workbooks](http://img.youtube.com/vi/7nWP_YRzxHg/0.jpg)](https://www.youtube.com/watch?v=7nWP_YRzxHg "Video showing how to combine data from different sources in workbooks")
+
+### Retrieving Alert data from Azure Resource Manager
+
+[![Image showing a grid with alerts data](../Images/ARMAlertsQueryGridFormat.png)](AlertDataARM.md)
+
+[See intrstructions](AlertDataARM.md)
```
Comment (yves-chan) [inline Documentation/Samples/Samples.md]: small typo here

File: Documentation/DataSources/DataSources.md [hunk 4]
```diff
@@ -12,6 +12,8 @@ Workbooks support these data sources:
 * [Workload Health](#workload-health)
 * [Azure Resource Health](#azure-resource-health)
 * [Custom Endpoint](#custom-endpoint)
+* [Azure Data Explorer](#azure-data-explorer)
+* [Azure Resource Manager](#azure-resource-manager)
```
Comment (acearun) [inline Documentation/DataSources/DataSources.md]: Should we include custom provider docs?

File: Documentation/DataSources/DataSources.md [hunk 5]
```diff
@@ -80,13 +82,23 @@ To avoid automatically making calls to un-trusted hosts when using templates, th
...
+## Azure Resource Manager (preview)
+Workbook supports Azure Resource Manager (ARM) REST operations. This allows the ability to query management.azure.com endpoint without the need to provide your own authorization header token.
+
+To make a query control use this data source, use the _Data source_ drop down to choose _Azure Resource Manager_. Provide the appropriate parameters such as Http method, url path, headers, url parameters and/or body.
+
+`Note: Only GET, POST, and HEAD operations are currently supported.`
+
+![A image of a workbook with an Azure Resource Manager query](../Images/AzureResourceManagerDataSource.PNG)
+
+
```
Comment (acearun) [inline Documentation/DataSources/DataSources.md]: Have a link here to the sample and transformation docs.

File: Documentation/Transformations/JSONPath.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# JSON Path
```
Comment (acearun) [inline Documentation/Transformations/JSONPath.md]: Let's also link to this page from JSON, Custom Endpoints and Custom Provider docs.

File: Documentation/DataSources/DataSources.md (hunk 4 above)
Comment (ericajlin) [reply Documentation/DataSources/DataSources.md]: Let's talk about what example we can provide. I can add this as a separate PR

**Pull request merged**